storage/atsd/atsd.go

```diff
@@ -0,0 +1,400 @@
+		atsdHttp.NewMetric(containerNetworkTcp6StatClosing).SetDataType(atsdHttp.LONG),
+
+		atsdHttp.NewMetric(containerTaskStatsNrIoWait).SetDataType(atsdHttp.LONG),
+		atsdHttp.NewMetric(containerTaskStatsNrRunning).SetDataType(atsdHttp.LONG),
+		atsdHttp.NewMetric(containerTaskStatsNrSleeping).SetDataType(atsdHttp.LONG),
+		atsdHttp.NewMetric(containerTaskStatsNrStopped).SetDataType(atsdHttp.LONG),
+		atsdHttp.NewMetric(containerTaskStatsNrUninterruptible).SetDataType(atsdHttp.LONG),
+
+		atsdHttp.NewMetric(containerDiskIoIoMerged).SetDataType(atsdHttp.LONG),
+		atsdHttp.NewMetric(containerDiskIoIoQueued).SetDataType(atsdHttp.LONG),
+		atsdHttp.NewMetric(containerDiskIoIoServiceBytes).SetDataType(atsdHttp.LONG),
+		atsdHttp.NewMetric(containerDiskIoIoServiced).SetDataType(atsdHttp.LONG),
+		atsdHttp.NewMetric(containerDiskIoIoServiceTime).SetDataType(atsdHttp.LONG),
+		atsdHttp.NewMetric(containerDiskIoIoTime).SetDataType(atsdHttp.LONG),
+		atsdHttp.NewMetric(containerDiskIoIoWaitTime).SetDataType(atsdHttp.LONG),
+		atsdHttp.NewMetric(containerDiskIoSectors).SetDataType(atsdHttp.LONG),
+
+		atsdHttp.NewMetric(containerFilesystemIoInProgress).SetDataType(atsdHttp.LONG),
+		atsdHttp.NewMetric(containerFilesystemIoTime).SetDataType(atsdHttp.LONG),
+		atsdHttp.NewMetric(containerFilesystemLimit).SetDataType(atsdHttp.LONG),
+		atsdHttp.NewMetric(containerFilesystemReadsCompleted).SetDataType(atsdHttp.LONG),
+		atsdHttp.NewMetric(containerFilesystemReadsMerged).SetDataType(atsdHttp.LONG),
+		atsdHttp.NewMetric(containerFilesystemReadTime).SetDataType(atsdHttp.LONG),
+		atsdHttp.NewMetric(containerFilesystemSectorsRead).SetDataType(atsdHttp.LONG),
+		atsdHttp.NewMetric(containerFilesystemSectorsWritten).SetDataType(atsdHttp.LONG),
+		atsdHttp.NewMetric(containerFilesystemUsage).SetDataType(atsdHttp.LONG),
+		atsdHttp.NewMetric(containerFilesystemBaseUsage).SetDataType(atsdHttp.LONG),
+		atsdHttp.NewMetric(containerFilesystemAvailable).SetDataType(atsdHttp.LONG),
+		atsdHttp.NewMetric(containerFilesystemInodesFree).SetDataType(atsdHttp.LONG),
+		atsdHttp.NewMetric(containerFilesystemWeightedIoTime).SetDataType(atsdHttp.LONG),
+		atsdHttp.NewMetric(containerFilesystemWritesCompleted).SetDataType(atsdHttp.LONG),
+		atsdHttp.NewMetric(containerFilesystemWritesMerged).SetDataType(atsdHttp.LONG),
+		atsdHttp.NewMetric(containerFilesystemWriteTime).SetDataType(atsdHttp.LONG),
+
+		atsdHttp.NewMetric(containerCpuUsageSystemPct).SetDataType(atsdHttp.FLOAT),
+		atsdHttp.NewMetric(containerCpuUsageTotalPct).SetDataType(atsdHttp.FLOAT),
+		atsdHttp.NewMetric(containerCpuUsageUserPct).SetDataType(atsdHttp.FLOAT),
+
+		atsdHttp.NewMetric(containerCpuHostUsageSystemPct).SetDataType(atsdHttp.FLOAT),
+		atsdHttp.NewMetric(containerCpuHostUsageTotalPct).SetDataType(atsdHttp.FLOAT),
+		atsdHttp.NewMetric(containerCpuHostUsageUserPct).SetDataType(atsdHttp.FLOAT),
+		atsdHttp.NewMetric(containerCpuUsagePerCpuPct).SetDataType(atsdHttp.FLOAT),
```

Comment:
I think this would be more readable if you iterated over a map instead. I.e.
```
for metric, dataType := range map[string]atsdHttp.DataType{
  containerCpuHostUsageSystemPct: atsdHttp.FLOAT,
  containerCpuHostUsageTotalPct: atsdHttp.FLOAT,
  ...
} {
  self.innerStorage.RegisterMetric(
    atsdHttp.NewMetric(metric).SetDataType(dataType))
}
```